resources/language/resource.language.nl_nl/strings.po

```diff
@@ -710,11 +710,11 @@ msgstr "Open de Up Next add-on instellingen"
 
 msgctxt "#30179"
 msgid "Show trailers & more"
-msgstr "Toon voorfilmpjes"
+msgstr "Toon trailers & meer"
```

Comment:
Never use ampersand ! Unless it is part of a name or title. Better use "en".
BTW This also applies to English.